src/regress_stack/__main__.py

```diff
@@ -122,16 +123,23 @@ def test():
     regress_list = pathlib.Path(dir_name) / "regress_tests.txt"
     regress_list.write_text(regress_tests)
 
-    try:
-        utils.run(
-            "tempest",
-            ["run", "--load-list", str(regress_list.relative_to(dir_name)), "--serial"],
-            env=env,
-            cwd=dir_name,
-        )
-    except subprocess.CalledProcessError:
-        # silence to fail on the next command
-        pass
+    # The tempest run is a long-running process and to improve UX we want
+    # direct output of both STDOUT and STDERR.
+    #
+    # Implementing that with subprocess is complicated, and as we do not need
+    # to process the output we can use os.system() directly.
+    saved_env = os.environ
+    os.environ.update(env)
+    load_list = str(regress_list.relative_to(dir_name))
+    os.system(
+        "bash -c '"
+        f"pushd {dir_name} && "
+        f"tempest run --load-list {load_list} --concurrency {concurrency} ; "
+        "popd"
+        "'"
+    )
```

Comment:
I think it would be simpler to go with not to fiddle with bash scripting from withing an os.system call, what do you think?

```suggestion
    cur_dir = os.getcwd()
    os.chdir(dir_name)
    try:
      os.system(f"tempest run --load-list {load_list} --concurrency {concurrency}")
    finally:
      os.chdir(cur_dir)
```

Reply:
That is a fair comment. From my perspective the bash fiddling is more clear, but I'll happily go with your suggestion

Reply:
`os.system` does not raise though, so I'll skip the try / finally:
```
>>> import os
>>> rc = os.system("non-existent")
sh: 1: non-existent: not found
>>> print(rc)
32512
```

Reply:
And might a well move to `core.utils` as a higher level helper.